mlflow/exceptions.py

```diff
@@ -33,6 +36,22 @@ def serialize_as_json(self):
         exception_dict.update(self.json_kwargs)
         return json.dumps(exception_dict)
 
+    def get_http_status_code(self):
+        http_error_map = {
```

Comment:
Small nit: Could we lift this as a top level module constant?

Reply:
Yup!